### DISCUSSION TITLE
Sudden Change in Brightness when Using Lower Brightness with Full Screen Video Playback

### DISCUSSION THREAD

**Discussion by TonyXiChen** [MonitorControl/MonitorControl#1388] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

I am using combined software and hardware dimming on my ASUS XG27U. When using full screen video playback with Safari or Chrome with brightness that triggers the software dimming (as in not natively supported by the hardware, I guess), if I launch mission control or swipe between desktops and swipe back, the monitor will flash a bit from the lowest hardware brightness to the software dimming one (which can be almost totally dark).

### Steps to reproduce

As described.

1. Safari or Chrome
2. Full Screen Video Playback
3. Software and Hardware Dimming Enabled
4. Brightness Set to Lower than the Lowest Supported Natively by the Monitor
5. Launch Mission Control or Swipe Between Desktops
6. Swipe Back to the Full Screen Video Playback

### Expected behavior

Consistent brightness.

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.3.1 Ventura
- Mac model: MacBook Pro (16-inch, 2021)
- MonitorControl version: 4.1.0 Build 7034
- Monitor(s): ASUS ROG XG27U
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment, marked as answer]
This is normal if you have gamma table manipulation disabled (Settings/Dispalys/Avoid gamma table manipulation) as then an overlay based software dimming is in effect which can't take place between mission control/desktop swipe transitions. This is a macOS limitation.
